docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -97,6 +98,15 @@ host_rewrite
   *(optional, string)* Indicates that during forwarding, the host header will be swapped with this
   value.
 
+.. _config_http_conn_man_route_table_route_auto_host_rewrite:
```

Comment:
Yes, that is a todo on my end. Need to provide an example as well. Will add it to router arch